packages/hoppscotch-common/src/components/environments/my/Environment.vue

```diff
@@ -150,6 +164,33 @@ const removeEnvironment = () => {
   toast.success(`${t("state.deleted")}`)
 }
 
+const environmentJson = () => {
+  const { ...newEnvironment } = props.environment
+  delete newEnvironment.id
+
+  return props.environmentIndex !== null
+    ? JSON.stringify(newEnvironment, null, 2)
+    : undefined
+}
+const exportJSON = () => {
+  const dataToWrite = environmentJson()
+  if (!dataToWrite) return
+  const file = new Blob([dataToWrite], { type: "application/json" })
+  const a = document.createElement("a")
+  const url = URL.createObjectURL(file)
+  a.href = url
+
+  // TODO: get uri from meta
+  a.download = `${url.split("/").pop()!.split("#")[0].split("?")[0]}.json`
```

Comment:
Could you elaborate on what's happening here? A comment will be useful as well. Also, can it be simplified?